apis/apps/v1alpha1/clusterdefinition_types.go

```diff
@@ -656,13 +651,6 @@ type ClusterComponentDefinition struct {
 	// +optional
 	VolumeTypes []VolumeTypeSpec `json:"volumeTypes,omitempty"`
 
-	// Used for custom label tags which you want to add to the component resources.
-	//
-	// +listType=map
-	// +listMapKey=key
-	// +optional
-	CustomLabelSpecs []CustomLabelSpec `json:"customLabelSpecs,omitempty"`
```

Comment:
Should the structure of CustomLabelSpec also be removed synchronously?

Reply:
done